test/server/cards/02_SHD/units/OmegaPartOfTheSquad.spec.ts

```diff
@@ -0,0 +1,65 @@
+describe('Omega, Part of the Squad', function() {
+    integration(function(contextRef) {
+        describe('Omega\'s Ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['omega#part-of-the-squad'],
+                        deck: ['crosshair#following-orders', 'pyke-sentinel', 'atst', 'cartel-spacer', 'battlefield-marine'],
+                        base: 'echo-base',
+                        leader: 'hera-syndulla#spectre-two'
+                    }
+                });
+            });
+
+            it('can draw a clone', function () {
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.omega);
+                expect(context.player1).toHavePrompt('Select a card to reveal');
+                expect(context.player1).toHaveEnabledPromptButton(context.crosshair.title);
+                expect(context.player1).toHaveDisabledPromptButtons([context.atst.title, context.battlefieldMarine.title, context.cartelSpacer.title, context.pykeSentinel.title]);
+                context.player1.clickPrompt(context.crosshair.title);
+                expect(context.crosshair).toBeInLocation('hand');
+                expect(context.getChatLogs(2)).toContain('player1 takes Crosshair');
+
+                // Omega should cost 4 since it cannot discount itself
```

Comment:
Here's a fun test - can we add a case where we play an Omega and then next phase play another copy of Omega and confirm that the second one gets the discount. I found some pretty clean patterns for getting references to two copies of a card during test setup, can see in `UniqueRule.spec.ts`